labs/jack-in/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "jack-in"
+description = "an experimental sliding sync/syncv3 terminal client to jack into the matrix"
+version = "0.2.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+file-logging = ["dep:log4rs"]
+
+[dependencies]
+tuirealm = "~1.7.1"
```

Comment:
Why the stricter version bound, rather than just `1.7.1` (no `~`)?

Reply:
because tuirealm 1.8 pulls in tui 0.19, while tui-logger in its latest version is build against tui 0.18. with `tuirealm = 1.7` it cargo picks `1.8` and then compile fails.